.gitignore

```diff
@@ -18,6 +21,9 @@ preview-build-message.txt
 packages/*/coverage
 packages/*/dist
 packages/*/docs
+apps/*/coverage
+apps/*/dist
+apps/*/docs
```

Comment:
Do these folders exist in apps? I don't see them in storybook-react-native can they be removed?

If you run `find apps -type d | grep -E '/(coverage|dist|docs)$'` in the terminal there are no results. 

![Screenshot 2024-10-22 at 1 01 35 PM](https://github.com/user-attachments/assets/05f02ece-2042-429c-956b-d3a1cdfd4be1)

If you run `find apps -type d | grep -E '/(coverage|dist|android)$'` it will pick up the android directory

![Screenshot 2024-10-22 at 1 01 50 PM](https://github.com/user-attachments/assets/89f57d6a-5f2f-45a1-917b-d0e903d652a8)


```suggestion
```